src/test/java/bio/terra/service/snapshot/flight/create/CreateSnapshotByRequestIdInterfaceTest.java

```diff
@@ -69,15 +71,16 @@ void prepareAndCreateSnapshot() throws InterruptedException {
     SnapshotRequestModel requestModel =
         SnapshotBuilderTestData.createSnapshotRequestByRequestId(snapshotAccessRequestId);
 
-    when(snapshotRequestDao.getById(snapshotAccessRequestId)).thenReturn(accessRequestResponse);
+    when(snapshotService.getSnapshotAccessRequestById(snapshotAccessRequestId))
+        .thenReturn(accessRequestResponse);
     when(snapshotDao.retrieveSnapshot(sourceSnapshotId)).thenReturn(sourceSnapshot);
     when(snapshotBuilderService.generateRowIdQuery(
             accessRequestResponse, sourceSnapshot, TEST_USER))
         .thenReturn(sqlQuery);
 
     // mock out call to createSnapshotPrimaryData
     when(createSnapshotByRequestIdInterface.createSnapshot(
-            eq(flightContext), eq(asset), eq(snapshot), eq(sqlQuery), any()))
+            any(), any(), eq(snapshot), eq(sqlQuery), any()))
```

Comment:
Is it no longer possible to derive `flightContext` or `asset`?

Reply:
I added back flightContext, but the asset is now generated in the call so I don't think I can set it here.